crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -297,6 +309,7 @@ impl Room {
             ))),
             room_info_notable_update_sender,
             seen_knock_request_ids_map: SharedObservable::new_async(None),
+            room_member_updates_sender: broadcast::channel(10).0,
```

Comment:
Can you use local variables instead? It's really not clear what the 0 is, so before assigning the sender here, you'd have a line above like `let (sender, _receiver) = broadcast::channel(10);`, and then use `sender` here.